docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -ex
+
+chown -R $UID:$GID $DATA_DIR
```

Comment:
no recursive chown in entrypoint - the code repository should already have permissions set in the Dockerfile

For when the code directory is mounted from liquid's `mount_local_repos`, we don't want to chown anything as the user won't be able to edit it outside the container anymore. The code should still execute (as per default permissions 755), so don't touch it at all